doc/Design.md

```diff
@@ -50,11 +50,11 @@ mind. Tock favors overall reliability of the system and discourages components
 
 ![Tock architecture](architecture.png)
 
-Tock includes three architectural components. A small trusted kernel, written in
-Rust, implements a hardware abstraction layer (HAL), scheduler and
+Tock includes three architectural components: a small trusted kernel, written in
+Rust, which implements a hardware abstraction layer (HAL), scheduler, and
```

Comment:
Because one of these list items contains a comma, the list should be semicolon-delimited:

Tock includes three architectural components: a small trusted kernel, written in Rust, which implements a hardware abstraction layer (HAL); scheduler; and platform-specific configuration.